stream-chat-android-compose/src/test/kotlin/io/getstream/chat/android/compose/viewmodel/imagepreview/ImagePreviewViewModelTest.kt

```diff
@@ -67,7 +68,7 @@ internal class ImagePreviewViewModelTest {
             viewModel.deleteCurrentImage(attachment1)
             viewModel.deleteCurrentImage(attachment2)
 
-            verify(chatClient).updateMessage(any())
+            verify(chatClient).updateMessage(any(), eq(false))
```

Comment:
Could we add this boolean value to our SUT?
We would need two tests to check both conditions (It could be a parametrized test)